### PR TITLE
Update Readme and Add sample gcode test file.

### PR DESCRIPTION
**Pull request by sithlord48**
Added a simple gcode file for users to attempt to print. should work on all firmwares and provide us with some basic infromation on the printer used. as well as doing some movement that the user will see to confirm it does work.

Signed-off-by: sith lord <sithlord48@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
-KPS - KDE Print Service for 3DPrinters [![Build Status](https://travis-ci.org/TeamWho/KPS.svg?branch=master)](https://travis-ci.org/TeamWho/KPS)
+AtCore - KDE Print Service for 3DPrinters [![Build Status](https://travis-ci.org/TeamWho/KPS.svg?branch=master)](https://travis-ci.org/TeamWho/KPS)
```
Comment (patrickelectric) [inline README.md]: Need to change travis to AtCore

File: README.md [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
 ----------------------------------------------------
 
-KPS is a API to manage the serial connection between the computer and the 3DPrinter.
+AtCore is a API to manage the serial connection between the computer and the 3DPrinter.
 
 >> It's currently in developing! <<
 
 This project is under LGPL V3, for more information read LICENSE
+
+Testing AtCore:
```
Comment (patrickelectric) [inline README.md]: add ### (or something) before Testing AtCore. ### Testing AtCore

File: README.md (hunk 1 above)
Comment (sithlord48) [inline README.md]: i tried that it didn't work.. do we need to change something in travis to AtCore first?

File: README.md (hunk 2 above)
Comment (sithlord48) [inline README.md]: added

**Pull request closed without merging**